Directory.Packages.props

```diff
@@ -6,6 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageVersion Include="Basic.CompilerLog.Util" Version="0.9.9" />
```

Comment:
I think this dependency is going to be a problem for source build - well either need to get it added to source build externals or find another way around the testing problem.

Reply:
No. Tests don't get source-built and can use external dependencies. cc @dotnet/source-build

Reply:
There is no plan to change it.  With the VMR, the MSFT build configuration will including building the tests but will not run them.  That will remain as part of the repo CI.

Reply:
so broadly, the source-build/VMR partners are counting on our non-VMR builds of the SDK for testing, and they've accepted that (or have other methods outside of the repo to do their own validation testing)?

Reply:
Correct. We accept that the VMR can't replace the rich test suites that run in the individual repositories. We do have some gates, i.e. http://github.com/dotnet/scenario-tests which is an E2E test suite that we expect will become more important going forward.